src/Tabs/TabList.tsx

```diff
@@ -1,25 +1,22 @@
+/* @jsx jsx */
+import { jsx } from '@emotion/core';
 import * as React from 'react';
 
-import { Dictionary } from '@stoplight/types';
 import { TabList, TabListProps } from 'react-tabs';
-import { IBoxCSS, useTheme } from '..';
 
 const StyledTabList: React.FunctionComponent<TabListProps> & { tabsRole: string } = props => (
-  <TabList {...props} css={tabListStyle()}>
+  // @ts-ignore ref type is incompatible after @types/react bump
```

Comment:
Yeah another one — this will propagate to other components, I suggest to revert the bump